metric-anomaly-data-model/src/main/avro/MetricAnomalyNotificationEvent.avdl

```diff
@@ -5,5 +5,11 @@ protocol MetricAnomalyNotificationEventProtocol {
     string event_condition_type;
     long violation_timestamp;
     string channel_id;
+    array<MetricValues> metric_values_list;
+  }
+
+  record MetricValues {
```

Comment:
Now, I am having a second thought, if the condition will be for the last 5 min or so, should we send all the values?
or the below message will be fine.

`the observed value for last 1 min was higher than X` which we can easily derive for condition information itself.
what do you think @rish691 ?

Reply:
Lets also add number of data point evaluated in the window

Reply:
rename this to ViolationSummary